test/presenters/statistics_announcement_presenter_test.rb

```diff
@@ -0,0 +1,128 @@
+require 'test_helper'
+
+class StatisticsAnnouncementPresenterTest < ActiveSupport::TestCase
+
+  include ActionView::Helpers::UrlHelper
+
+  test 'presents from as links to organisations' do
+    item = presented_statistics_announcement({
+      "links" => {
+        "organisations" => [{ "title" => 'Org', "base_path" => '/orgs/org'}]
+      }
+    })
+    links = [
+      link_to('Org', '/orgs/org')
+    ]
+    assert_equal links, item.from
+  end
+
+  test 'presents part_of as links to a topic' do
+    item = presented_statistics_announcement({
+      "links" => {
+        "policy_areas" => [{ "title" => 'Policy Area', "base_path" => '/orgs/policy-area'}]
+      }
+    })
+    links = [
+      link_to('Policy Area', '/orgs/policy-area')
+    ]
+    assert_equal links, item.part_of
+  end
+
+  test 'presents release_date' do
+    item = presented_statistics_announcement({
+      "details" => {
+        "display_date" => "About Midday on Tuesday"
```

Comment:
As discussed on Slack, it would be good to see this be a more realistic example, like “April to May 2016”